source/features/parse-backticks.tsx

```diff
@@ -15,18 +15,28 @@ function initGeneral(): void {
 		'[id^=ref-issue-]', // Issue references in `isIssue`, `isPRConversation`
 		'[id^=ref-pullrequest-]', // PR references in `isIssue`, `isPRConversation`
 		'.TimelineItem-body > del, .TimelineItem-body > ins', // Title edits in `isIssue`, `isPRConversation`
-		'.js-pinned-issue-list-item > span' // Pinned Issues
+		'.js-pinned-issue-list-item > span', // Pinned Issues
+		'blame-commit-message a' // `isBlame`
 	])) {
 		parseBackticks(title);
 	}
 }
 
+export function parseBackticksOnNewsFeedItems(): void {
+	for (const title of select.all('.issues_labeled .text-gray-dark > a:not(.rgh-parse-backticks), .commits blockquote:not(.rgh-parse-backticks)')) {
```

Comment:
`.commits blockquote:not(.rgh-parse-backticks)`
This is the push commits if clean dashbaord is off

Reply:
I'd
1. Merge this info `initGeneral`'s array
2. Rename `initGeneral` to `parseAll` or something
3. Add `.map(selector => selector + ':not(.rgh-parse-backticks)') to that array
4. Actually add the class inside the loop

More items in that list are dynamic like this and may benefit from being called `onNewComments`

Alternatively you could have 2 separate functions:

- `parseStatic`: no need for `.rgh-parse-backticks` class
- `parseDynamic`: called by the newsfeed listener and onNewComments (not necessarily in this PR)

Reply:
The issue is that we need to wait for it to load just like the other dashboard events

Reply:
I started doing this and I realized that this may be over my head. (Unless you want to guide me)